src/components/game/game.test.jsx

```diff
@@ -1,46 +1,52 @@
 import React from 'react';
-import Game from './game';
 import { shallow, mount } from 'enzyme';
+import Game from './game';
 
-it('<Game/> renders without error', () => {
-  shallow(<Game />);
-});
 
-it('<Game/> status is correct', () => {
-  const wrapper = mount(<Game />);
-  const firstPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(firstPlayer).toEqual('Next player is X');
+describe('Game', () => {
+  it('renders without error', () => {
+    shallow(<Game />);
+  });
 
-  const button = wrapper
-    .find('button.square')
-    .first()
-    .simulate('click');
-  const secondPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(secondPlayer).toEqual('Next player is O');
+  it('status is correct', () => {
+    const comp = mount(<Game />);
```

Comment:
Why the preference of `mount` as opposed to `shallow` here?

Reply:
I was referring to the component itself but I see now that using shallow would be preferred as I am testing something that will be rendered.